### PR TITLE
Feature/vih 4432 access to room representative

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-4432

### Change description ###

create page access to a room for represantative

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.vh-border {
```
Comment (ddikman) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.css]: are these specific to the `access to a room` or can they be moved up to the general css?

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.ts [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 
 @Component({
   selector: 'app-access-to-room',
   templateUrl: './access-to-room.component.html',
-  styles: []
+  styleUrls: ['./access-to-room.component.css']
 })
-export class AccessToRoomComponent extends RepresentativeBaseComponent {
+export class AccessToRoomComponent extends SuitabilityChoicePageBaseComponent {
+
```
Comment (ddikman) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.ts]: I know we're missing this on other pages as well but maybe it's better we start putting it in now to start with:
on ngInit can you set the choice value to the model room value?

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.css (hunk 1 above)
Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.css]: Yes, it's specific

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.ts (hunk 2 above)
Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/access-to-room/access-to-room.component.ts]: But it was set in bindModel() ? or do you mean something else?

**Pull request merged**